README.md

```diff
@@ -209,6 +209,8 @@ get_current_span().add_event('<error-message>', {'lumigo.type': '<error-type>'})
 | | | 0.100.0b2~0.100.0b3 |
 | | [uvicorn](https://pypi.org/project/uvicorn) | 0.11.3~0.22.0 |
 | flask | [flask](https://pypi.org/project/flask) | 2.0.0~2.2.5 |
+| kafka_python | [kafka_python](https://pypi.org/project/kafka_python/) | 2.0.0~2.0.2 |
```

Comment:
this is just to serve as a reminder to add the grpcio versions once testing is complete, just replace this when those tests are ready

Reply:
@saartochner  i've updated the grpcio versions to include 1.56.0 (which is the only version being tested at present), let me know if we should have more in there...

Reply:
we're now testing the last year's versions successfully 👍